br/pkg/stream/stream_status.go

```diff
@@ -117,9 +117,9 @@ func (p *printByTable) AddTask(task TaskStatus) {
 	table := p.console.CreateTable()
 	table.Add("name", task.Info.Name)
 	table.Add("status", task.colorfulStatusString())
-	table.Add("start", fmt.Sprint(oracle.GetTimeFromTS(task.Info.StartTs)))
+	table.Add("start", fmt.Sprint(oracle.GetTimeFromTS(task.Info.StartTs).Format("2006-01-02 15:04:05.999999999 -0700")))
```

Comment:
maybe use a common function and add a unit test case to make sure the output of this common function.

Reply:
Done. PTAL